static/js/directives/ui/quota-management-view.js

```diff
@@ -295,6 +302,10 @@ angular.module('quay').directive('quotaManagementView', function () {
         $scope.newLimitConfig = {"limit_percent": 100, "type": $scope.rejectLimitType};
       }
 
+      $scope.isObjectEmpty = function(obj){
```

Comment:
newly defined but I dont see it used.

Reply:
I am using it [here](https://github.com/quay/quay/pull/1280/files#diff-cff2e8df5bd2e9560c90e44d2019f0fb2987c47f2dc89e753123e39262398f8eR33)